.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,52 @@
+name: UnitTests
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+
+jobs:   
+  run-unit-tests:
+
+    runs-on: ubuntu-latest
+    container: compucorp/civicrm-buildkit:1.0.0
+    
+    env:
+      REPOSITORY_NAME: ${{ github.event.repository.name }}
+      CIVICRM_CORE: 5.24.6
```

Comment:
I guess the docker image has to be updated from time to time anytime we update civicrm core in compuclient? Would we have to update this line then to reflect the latest civicrm version ?

Reply:
@tunbola  the docker image will not need to be updated from time to time but workflow need to be updated when CiviCRM Core has been updated in compuclient.